datacapture/src/androidTest/java/com/google/android/fhir/datacapture/views/QuestionnaireItemDropDownViewHolderFactoryEspressoTest.kt

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.android.fhir.datacapture.views
+
+import android.view.View
+import android.widget.FrameLayout
+import android.widget.TextView
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.RootMatchers
+import androidx.test.espresso.matcher.ViewMatchers.isDisplayed
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.espresso.matcher.ViewMatchers.withText
+import androidx.test.ext.junit.rules.ActivityScenarioRule
+import androidx.test.platform.app.InstrumentationRegistry
+import com.google.android.fhir.datacapture.R
+import com.google.android.fhir.datacapture.TestActivity
+import com.google.android.fhir.datacapture.utilities.showDropDown
+import com.google.common.truth.Truth
+import org.hl7.fhir.r4.model.Coding
+import org.hl7.fhir.r4.model.Questionnaire
+import org.hl7.fhir.r4.model.QuestionnaireResponse
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+
+class QuestionnaireItemDropDownViewHolderFactoryEspressoTest {
+  @Rule
+  @JvmField
+  var activityScenarioRule: ActivityScenarioRule<TestActivity> =
+    ActivityScenarioRule<TestActivity>(TestActivity::class.java)
+
+  private lateinit var parent: FrameLayout
+  private lateinit var viewHolder: QuestionnaireItemViewHolder
+
+  @Before
+  fun setup() {
+    activityScenarioRule.getScenario().onActivity { activity -> parent = FrameLayout(activity) }
+    viewHolder = QuestionnaireItemDropDownViewHolderFactory.create(parent)
+    setTestLayout(viewHolder.itemView)
+  }
+
+  @Test
+  fun shouldClearAutoCompleteTextView() {
+    val questionnaireItemViewItem =
+      QuestionnaireItemViewItem(
+        answerOptions("Coding 1", "Coding 2", "Coding 3", "Coding 4", "Coding 5"),
+        responseOptions()
+      ) {}
+    runOnUI { viewHolder.bind(questionnaireItemViewItem) }
+
+    onView(withId(R.id.auto_complete)).perform(showDropDown())
+    onView(withText("-"))
+      .inRoot(RootMatchers.isPlatformPopup())
+      .check(matches(isDisplayed()))
+      .perform(ViewActions.click())
+    Truth.assertThat(viewHolder.itemView.findViewById<TextView>(R.id.auto_complete).text.toString())
```

Comment:
import `Truth.assertThat`

```suggestion
   assertThat(viewHolder.itemView.findViewById<TextView>(R.id.auto_complete).text.toString())
```

Reply:
Done